spec/acceptance/puppetserver_upgrade_6_7_0_to_6_7_2_spec.rb

```diff
@@ -0,0 +1,90 @@
+require 'spec_helper_acceptance'
+
+describe 'Scenario: 6.7.0 to 6.7.2 upgrade:', if: ENV['BEAKER_PUPPET_COLLECTION'] == 'puppet6' do
+  before(:context) do
+    if check_for_package(default, 'puppetserver')
+      on default, puppet('resource package puppetserver ensure=purged')
+      on default, 'rm -rf /etc/sysconfig/puppetserver /etc/puppetlabs/puppetserver'
+      on default, 'rm -rf /etc/puppetlabs/puppet/ssl'
+    end
+
+    # puppetserver won't start with lower than 2GB memory
+    memoryfree_mb = fact('memoryfree_mb').to_i
+    raise 'At least 2048MB free memory required' if memoryfree_mb < 256
```

Comment:
This looks odd. Should this be total types 2GB instead of 256 MB free?

Reply:
Reworded the comments. This was copied over from the Puppet 5 test.